### ISSUE TITLE
Changed signature in MacOS version 2.4.0? 

### ISSUE THREAD

**Issue by suschizu** [mixxxdj/mixxx#12825]
### Bug Description

EXPECTED 'RJ Skerry-Ryan (FLYL4D545V)' 
FOUND 'Daniel Schuermann (JBLRSP95FC)'

### Version

2.4.0

### OS

macOS 13.6

**Comment by Swiftb0y** [comment]
Yes, we changed some infrastructure around. The change in code signature is expected. How did you get this error?
